### ISSUE TITLE
 USE_EXACT_ALARM permission issue

### ISSUE THREAD

**Issue by yan88a** [kyoz/godot-local-notification#6]
### Godot version

4.1.1

### Plugin version

v1

### Phone information

Samsung S21

### Issue description

love your work.

seems google hardened their permission policy regarding the use of the alarm.
when submitting the app i get: 
Remove the use of USE_EXACT_ALARM permission from all version codes within the submission. This includes both production and testing tracks.
Consider using the SCHEDULE_EXACT_ALARM permission as an alternative option if you have a use case for exact alarm functionality.

any chance the  SCHEDULE_EXACT_ALARM will provide the same functionality?

thanks!

### Steps to reproduce

submit to google..

**Comment by kyoz** [comment]
Hello there, in fact in the past the plugin do have `USE_EXACT_ALARM` permission, but i do faced same problem when publish my app to so i have updated the plugin. Can you try the newest version, as well as checked that other plugins doesn't have `USE_EXACT_ALARM` permission?

**Comment by yan88a** [comment]
yep - i tried the latest version from Nov 26.. (release for 4.1.3)
to be honest i saw in the manifest you are using the com.android.alarm.permission.SET_ALARM
but for some reason - the guys at google interoperate it as the use exact alarm one..

**Comment by yan88a** [comment]
seems the only permission needed is "post notification"
https://developer.android.com/develop/ui/views/notifications/build-notification

will try to build one with it - wont build for some reason... in any case - i'll try to build something just with "post notification" permission - and test it out-  if it will work - i'll update here

**Comment by yan88a** [comment]
couldnt get it to compile properly for some reason. 
godot is throwing an error...

in any case - will try again this weekend.

seems that leaving only the "post notification" permission should do the trick

**Comment by kyoz** [comment]
My assume is that there is some other plugins or setting that have "USE_EXACT_ALARM" on it. `com.android.alarm.permission.SET_ALARM` it self doesn't have anything to do with that suggestion from google.

Make sure that you not checked `Use Alarm` on Godot export if it's not necessary.
![Screenshot 2024-03-27 at 16 43 59](https://github.com/kyoz/godot-local-notification/assets/29815830/f8cc3629-62d8-46cc-966b-2c52bf51b69c)

BTW, how many plugin do you use for your project. Can you list em out? I'll try do look into em.

**Comment by yan88a** [comment]
seems you are right- just tried to upload the app without the notification plugin and it was rejected.
also have your share plugin - but saw its not using that as well..

my guess - probably ad mob :)

will update for curiosity's sake :)
